Src/DependencyCollector/Shared/Implementation/FrameworkSqlEventListener.cs

```diff
@@ -47,8 +48,7 @@ private enum CompositeState
         /// Performs application-defined tasks associated with freeing, releasing, or resetting unmanaged resources.
         /// </summary>
         public override void Dispose()
```

Comment:
If the only thing in here is base.Dispose() then this override can just be deleted.